### PR TITLE
Crat

### PR DESCRIPTION
**Pull request by PrasannaKavedi**
MY first pull request

### CODE REVIEW

File: Donkey [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Domestic animal
+Ilove Donkey
```
Comment (TharYoke) [inline Donkey:2]: Its good idea to add donkeys in the git hub :)

Comment (PrasannaKavedi) [reply Donkey:2]: Thanks a lot for your comments

File: Lion [hunk 2]
```diff
@@ -0,0 +1 @@
+king
```
Comment (TharYoke) [inline Lion:1]: Lion King - WOW

**Review by TharYoke — COMMENTED** [review]
Very funny github

**Review by TharYoke — APPROVED** [review]
Great Job Pras

**Pull request merged**